java/src/jmri/jmrit/logixng/util/parser/functions/ConvertFunctions.java

```diff
@@ -27,6 +27,8 @@ public String getModule() {
     @Override
     public Set<Function> getFunctions() {
         Set<Function> functionClasses = new HashSet<>();
+        functionClasses.add(new IsIntFunction());
```

Comment:
(Not for this PR) Java 11 has a nice way to create an immutable set that can be used here:
```
Set<String> immutable = Set.of("Item 1", "item 2");
```